common/static/common/templates/discussion/thread-list-item.underscore

```diff
@@ -72,7 +72,18 @@
       %>
       </span>
 
-      <span class="forum-nav-thread-comments-count <% if (unread_comments_count > 0) { %>is-unread<% } %>">
+      <% if (!neverRead && unread_comments_count > 0) { %>
+        <span class="forum-nav-thread-unread-comments-count">
+          <%-
+            StringUtils.interpolate(
+              gettext('{unread_comments_count} new'),
+              {unread_comments_count: unread_comments_count}
```

Comment:
Nit: the JS styleguide says to infix space in single line objects:

```
{ unread_comments_count: unread_comments_count }
```

but we don't (can't?) lint `.underscore templates` - it would still be good to be consistent.